src/Components/Item/Item.scss

```
@@ -8,8 +8,8 @@
       padding: 0px 10px;
       .item_photo_box{
```

```suggestion
      padding: 0px 10px;
      
      .item_photo_box {
```
가독성을 위해서 새로운 셀렉터가 시작될 때 한줄 씩 띄워주시고 중괄호도 셀렉터와 한칸 띄워주세요

---

넵 팀원분 머지된 부분인 것 같은데, 같이 공유해서 신경쓰도록 하겠습니다.